packages/dds/tree/src/test/simple-tree/proxies.spec.ts

```diff
@@ -161,6 +169,31 @@ describe("SharedTreeObject", () => {
 		root.optional = undefined;
 		assert.equal(root.optional, undefined);
 	});
+
+	it("returns the stable id under the identifier field kind.", () => {
+		const schemaWithIdentifier = sb.object("parent", {
+			identifier: sb.identifier(),
+		});
+		const idCompressor = createIdCompressor();
```

Comment:
In your treeApi.spec.ts test, you constructed a mock nodeKeyManager rather than directly creating an idCompressor. Is there any reason not to do that here as well?

Reply:
updated to use a mock nodeKeyManager to simplify test